app/konnect/service-catalog/index.md

```diff
@@ -0,0 +1,133 @@
+---
+title: Service Catalog
+subtitle: Track every service across your architecture
+content-type: explanation
+---
+
+{{site.konnect_saas}}'s Service Catalog offers a comprehensive catalog of all services running in your organization. This catalog gives you insight into your organization's services and their dependencies. It includes both {{site.base_gateway}} services and external integration resources.
+
+<!-- vale off-->
+{% mermaid %}
+graph LR
+  %% Define styles for icons and nodes
+  classDef integration fill:#f7f7f7,stroke:#333,stroke-width:2px;
+  classDef resource fill:#e1f7d5,stroke:#333,stroke-width:2px;
+  classDef konnect fill:#d0e5ff,stroke:#333,stroke-width:2px;
+  classDef serviceCatalog fill:#e1f7d5,stroke:#333,stroke-width:2px;
+  classDef billingInventory fill:#f0f0f0,stroke:#333,stroke-width:2px;
```

Comment:
Curious about the styling here. I think it's a great idea to group elements by colours (the green, for example, groups integrations and services) - but I'm not sure about what the rest of it is doing. `stroke:#333,stroke-width:2px;` seems to set all the borders to thick black lines - what's the goal of this? Were there contrast issues with the original diagram?